templates/rental-agreement/templates/main.tpl

```diff
@@ -0,0 +1,487 @@
+{{define "checklist-row" }}
```

Comment:
Probably need to make this consistent? like adding space after "{{" and before "}}", and this became `{{ define "checklist-row" }}